src/schemas.ts

```diff
@@ -33,36 +33,17 @@ export const recipientSchema = yup.object({
     .object({
       amountBreakdown: yup
         .object({
-          welcomeTravelerRewards: yup
-            .string()
-            .test(testIsPositiveBigNumberish)
-            .default("0"),
-          earlyUserRewards: yup
-            .string()
-            .test(testIsPositiveBigNumberish)
-            .default("0"),
+          welcomeTravelerRewards: yup.string().test(testIsPositiveBigNumberish),
+          earlyUserRewards: yup.string().test(testIsPositiveBigNumberish),
           liquidityProviderRewards: yup
             .string()
-            .test(testIsPositiveBigNumberish)
-            .default("0"),
-          communityRewards: yup
-            .string()
-            .test(testIsPositiveBigNumberish)
-            .default("0"),
-          referralRewards: yup
-            .string()
-            .test(testIsPositiveBigNumberish)
-            .default("0"),
-          opRewards: yup
-            .string()
-            .test(testIsPositiveBigNumberish)
-            .default("0"),
-          arbRewards: yup
-            .string()
-            .test(testIsPositiveBigNumberish)
-            .default("0")
+            .test(testIsPositiveBigNumberish),
+          communityRewards: yup.string().test(testIsPositiveBigNumberish),
+          referralRewards: yup.string().test(testIsPositiveBigNumberish),
+          opRewards: yup.string().test(testIsPositiveBigNumberish),
+          arbRewards: yup.string().test(testIsPositiveBigNumberish)
         })
-        .defined()
+        .notRequired()
```

Comment:
since the metadata object is going to be removed, maybe this validation logic can also be removed

Reply:
Removed here [51eae69](https://github.com/across-protocol/merkle-distributor/pull/15/commits/51eae69dea24965dbd14383fe0b8a28a869174d8)